documentation.md

```diff
@@ -0,0 +1,344 @@
+# Classroom Angel Internal API Documentation
+
+## Teacher Attendance
+
+- GET to [https://classroom-angel-development.herokuapp.com/api/attendance/](https://classroom-angel-development.herokuapp.com/api/attendance/) - retrieve all attendance records
+  URL Params | Body | Returns
+  --- | --- | ---
+  N/A | N/A | All attendance objects (return)
+
+  > Attendance object (return): {
+  > "id": integer,
+  > "last_in": integer,
+  > "last_out": integer,
+  > "total_minutes_missed": integer,
+  > "expected_hours": integer
+  > }
+
+- POST to [https://classroom-angel-development.herokuapp.com/api/attendance/](https://classroom-angel-development.herokuapp.com/api/attendance/) - create attendance record
+  URL Params | Body | Returns
+  --- | --- | ---
+  N/A | Attendance object (send) | Attendance object (return)
```

Comment:
This table looks like it isn't fully formatted!

Reply:
Done